add-stATOM-oracles-permit.json

```diff
@@ -6,6 +6,7 @@
     "chainTimerService": "priceFeed",
     "client": "priceFeed",
     "econCharterKit": "priceFeed",
+    "economicCommitteeCreatorFacet": "priceFeed",
```

Comment:
Note: check to see if we previously removed this. The current `agoric-sdk` build produces this output